src/js/helpers/xmppEventHandlers.js

```diff
@@ -236,20 +243,17 @@ module.exports = function (client, app) {
     });
 
     client.on('avatar', function (info) {
-        var contact = me.getContact(info.jid);
+        var contact = me.getContact(info.jid) || me.isMe(info.jid);
         if (!contact) {
-            if (me.isMe(info.jid)) {
-                contact = me;
-            } else {
-                return;
-            }
```

Comment:
I believe this change is incorrect. Here the `contact` should be equal `me` in case of `me.getContact(info.jid)` and `me.isMe(info.jid)`. But you put there the result of `me.isMe(info.jid)` (I believe it's a bool and not a contact).